src/shared_modules/content_manager/src/components/CtiDownloader.hpp

```diff
@@ -190,16 +220,47 @@ class CtiDownloader : public AbstractHandler<std::shared_ptr<UpdaterContext>>
             }
             catch (const cti_server_error& e)
             {
-                constexpr auto SLEEP_TIME_THRESHOLD {30};
-
-                logError(WM_CONTENTUPDATER, e.what());
+                logDebug1(WM_CONTENTUPDATER, e.what());
 
-                // Increase sleep time exponentially, up to the threshold
-                if (sleepTime < SLEEP_TIME_THRESHOLD)
+                switch (e.type())
                 {
-                    sleepTime = std::min(SLEEP_TIME_THRESHOLD, static_cast<int>(std::pow(2, retryAttempt)));
-                    ++retryAttempt;
+                    case CtiErrorType::GENERIC_SERVER_ERROR:
+                    {
+                        if (CtiErrorType::GENERIC_SERVER_ERROR == lastErrorType)
+                        {
+                            // Increase sleep time exponentially, up to the threshold
+                            constexpr auto SLEEP_TIME_THRESHOLD {30};
+                            if (sleepTime < SLEEP_TIME_THRESHOLD)
+                            {
+                                sleepTime = std::min(SLEEP_TIME_THRESHOLD, static_cast<int>(std::pow(2, retryAttempt)));
+                                ++retryAttempt;
+                            }
+                        }
+                        else
+                        {
+                            // First time with this particular error.
+                            constexpr unsigned int INITIAL_SLEEP_TIME {1};
```

Comment:
Maybe this constant could be declared in the class scope to make its definition more visible.

Reply:
Done. It came handy for the new tests added.